packages/regl-worldview/src/WorldviewContext.js

```diff
@@ -233,6 +234,7 @@ export class WorldviewContext {
   }
 
   _paint() {
+    this._needsPaint = false;
```

Comment:
Should we set `this._needsPaint=false` at the end of this function? I'm a little confused.

Reply:
The goal is to see if it was set to true before the paint finished

Reply:
ah, that makes sense. I didn't read your PR description carefully.